lib/chewy/type/adapter/mongoid.rb

```diff
@@ -17,7 +17,9 @@ def identify(collection)
       private
 
         def cleanup_default_scope!
-          Chewy.logger.warn('Default type scope order, limit and offset are ignored and will be nullified') if Chewy.logger && @default_scope.options.values_at(:sort, :limit, :skip).compact.present?
+          Chewy.logger.warn('Default type scope order, limit and offset are ignored and will be nullified') if Chewy.logger && @default_scope.options.values_at(
+            :sort, :limit, :skip
+          ).compact.present?
```

Comment:
Maybe here it would be better to have this

```ruby
if Chewy.logger && @default_scope.options.values_at(:sort, :limit, :skip).compact.present?
  Chewy.logger.warn('Default type scope order, limit and offset are ignored and will be nullified')
end
```

Even better would be extracting `@default_scope.options.values_at(:sort, :limit, :skip).compact.present?` to another method and having:


```ruby
if Chewy.logger && sort_or_limit_or_skip_options?
  Chewy.logger.warn('Default type scope order, limit and offset are ignored and will be nullified')
end

...
def sort_or_limit_or_skip_options?
  @default_scope.options.values_at(:sort, :limit, :skip).compact.present?
end
```